src/core/public/entry_point.ts

```diff
@@ -33,7 +33,11 @@ const injectedMetadata = JSON.parse(
   document.querySelector('kbn-injected-metadata')!.getAttribute('data')!
 );
 
-if (process.env.IS_KIBANA_DISTRIBUTABLE !== 'true' && process.env.ELASTIC_APM_ACTIVE === 'true') {
+/**
+ * If APM RUM agent is enabled while starting the server,
+ * `apmConfig` would be populated with relavant configuration
+ */
+if (injectedMetadata.vars.apmConfig != null) {
```

Comment:
Currently, even if we set `ELASTIC_APM_ACTIVE ` before starting server, the RUM agent will not be enabled as we don't have it on webpack bundles. We fallback to using the config instead of adding yet another ENV variable. 

Relevant details can be found here https://github.com/elastic/kibana/blob/c579b6718115fdfac7ad426545efb53c4d0ec9d9/src/legacy/ui/apm/index.js#L23-L36

Reply:
I think we should still include the `process.env.IS_KIBANA_DISTRIBUTABLE !== 'true'` check for dead code elimination in production.

Reply:
Good point. will add that.